js/base/Exchange.js

```diff
@@ -1207,6 +1220,10 @@ module.exports = class Exchange {
             'timeInForce': timeInForce,
             'postOnly': postOnly,
             'trades': trades,
+            'reduceOnly': this.safeValue (order, 'reduceOnly'),
+            'triggerPrice': this.parseNumber (this.safeString2 (order, 'triggerPrice', 'stopPrice')),
+            'status': this.safeString (order, 'status'),
+            'fee': this.safeNumber (order, 'fee'),
```

Comment:
`fee` is an object with `cost` and `currency` inside, why are you using `safeNumber` ?